gurubu-client/src/app/api/og/route.tsx

```diff
@@ -0,0 +1,53 @@
+import { ImageResponse } from "next/og";
+import { CSSProperties } from "react";
+
+export const runtime = "edge";
+
+const imageStyle: CSSProperties = {
+  backgroundColor: "#ffffff",
+  backgroundSize: "150px 150px",
+  height: "100%",
+  width: "100%",
+  display: "flex",
+  textAlign: "center",
+  alignItems: "center",
+  justifyContent: "center",
+  flexDirection: "column",
+  flexWrap: "nowrap",
+  fontSize: "48px",
+};
+
+const imageSize = {
+  width: 1200,
+  height: 630,
+};
+
+export async function GET(request: Request) {
+  try {
+    const imageData = await fetch(
+      new URL("../../../../public/logo-with-text.png", import.meta.url)
+    ).then((res) => res.arrayBuffer());
```

Comment:
is it possible to take it as an ordinary picture?

Reply:
@mertcanaltin actually I thought that actually. As I used the edge runtime for this operation, all resources have to get with fetch api. [This link](https://vercel.com/docs/functions/og-image-generation#edge-runtime) explains that as well. Of course, if I used Node runtime, we can get the data via `fs.readFile`.